### ISSUE TITLE
antergos-gfxboot new-old-background

### ISSUE THREAD

**Issue by killajoe** [Antergos/antergos-gfxboot#5]
color of text needs to be set to dark, as white text on almost white background is ... ;)

**Comment by karasu** [comment]
@killajoe hahaha... yes, yes, I know... this is not finished. I'm planning on change the background image as having a light one is not good for people with visual difficulties...

**Comment by killajoe** [comment]
I see this only by accident... antergos-iso testing seems to build gfx-boot directly from github? i do not find any related inside build.sh and config .... as i try to change boot menue ...

**Comment by karasu** [comment]
>antergos-iso testing seems to build gfx-boot directly from github?

Nope. You must clone antergos-gfxboot into it. Then, make it. Then you will be able to run build.sh

I will include this directly in build.sh, seems a good idea.

**Comment by killajoe** [comment]
> cloning antergos-iso > changing to testing branch > cloning antergos-gfxboot > changing configs (background and isolinux.cfg) > make antergos-gfx-boot > make install antergos-iso > building iso : 
Resulting in having tha backgrounds and config from default / original state of antergos-gfxboot

**Comment by killajoe** [comment]
as you say: "I'm planning on change the background image as having a light one is not good for people with visual difficulties..."

May asking at the forum if "our" wallpaper artists can do some backgrounds ?

**Comment by killajoe** [comment]
from translations.sh:

```
cp -RL "${_SCRIPT_DIR}/antergos-gfxboot" "/usr/share/gfxboot/themes/"
cd /usr/share/gfxboot/themes/antergos-gfxboot
tx pull
make
cp -R /usr/share/gfxboot/themes/antergos-gfxboot/isolinux "${_SCRIPT_DIR}"
```

**Comment by lots0logs** [comment]
Its a submodule...you have to run `git submodule update --init` after cloning antergos-iso

**Comment by karasu** [comment]
>Its a submodule...

I totally forgot about this.

**Comment by killajoe** [comment]
i do also get this on build:

`
/usr/share/antergos-iso/configs/gnome/translations.sh: line 31: tx: command not found.`

**Comment by lots0logs** [comment]
Install `transifex-client` with pacman...

**Comment by killajoe** [comment]
may overstressing this issue thread...
But what are the specifications to get the gfxboot backgrounds shown up ? if i change the backgrounds they never show up i got only an black background... i use exact the same res an size... is it about colour depth?

**Comment by karasu** [comment]
I don't remember... try jpg with 256 colors only (or open the existing background images and see for youself 😀 )

**Comment by killajoe** [comment]
as far as i can see it can be only filesize... as all data i can get of the files are the same.. :) i will try ...

**Comment by lots0logs** [comment]
Why don't we just change the color of the white text to something that is more readable?

**Comment by karasu** [comment]
Black text over white background is a no no for people with visual
difficulties

**Comment by karasu** [comment]
Also, the antergos title has rendering problems in that image. Imho, this
should be redone.

**Comment by killajoe** [comment]
.... Readme of antergos-gfxboot say how to convert the background images 🔎

**Comment by Rizwan-Hasan** [comment]
I am also facing background changing problem.It never shows up

**Comment by karasu** [comment]
All the graphics need to have some special bits set, else GFXBoot will show just black images.

Use the 'imagemagick' packages tools to create the files in the correct format.

Examples:
```
convert -resize 640 -quality 90 -sampling-factor 2x2 -interlace none my_background.jpg back640x480.jpg

convert -resize 800 -quality 90 -sampling-factor 2x2 -interlace none my_background.jpg back800x600.jpg
```
Note: Kolourpaint also saves jpeg images in the correct format.
